catalogue/webapp/pages/images.tsx

```diff
@@ -0,0 +1,283 @@
+      />
+    );
+  }
+
+  return (
+    <>
+      <Head>
+        {results?.type === 'ResultList' && results.prevPage && (
+          <link
+            rel="prev"
+            href={convertUrlToString(
+              imagesLink(
+                { ...imagesRouteProps, page: (page || 1) - 1 },
+                'meta_link'
+              ).as
+            )}
+          />
+        )}
+        {results?.type === 'ResultList' && results.nextPage && (
+          <link
+            rel="next"
+            href={convertUrlToString(
+              imagesLink({ ...imagesRouteProps, page: page + 1 }, 'meta_link')
+                .as
+            )}
+          />
+        )}
+      </Head>
+
+      <CataloguePageLayout
+        title={`${query ? `${query} | ` : ''}image search`}
+        description="Search Wellcome Collection images"
+        url={imagesLink({ ...imagesRouteProps }, 'canonical_link').as}
+        openGraphType={'website'}
+        jsonLd={{ '@type': 'WebPage' }}
+        siteSection={'collections'}
+        imageUrl={null}
+        imageAltText={null}
+      >
+        <Space
+          v={{
+            size: 'l',
+            properties: ['padding-bottom'],
+          }}
+          className={classNames(['row'])}
+        >
+          <div className="container">
+            <div className="grid">
+              <div className={grid({ s: 12, m: 12, l: 12, xl: 12 })}>
+                {searchPrototype ? (
+                  <>
+                    <SearchTabs
+                      worksRouteProps={imagesRoutePropsToWorksRouteProps(
+                        imagesRouteProps
+                      )}
+                      imagesRouteProps={imagesRouteProps}
+                      workTypeAggregations={[]}
+                      shouldShowImagesFilters={query !== ''}
+                      shouldShowWorksFilters={false}
+                      shouldShowDescription={query === ''}
+                      activeTabIndex={1}
+                    />
+                  </>
+                ) : (
+                  <SearchForm
+                    ariaDescribedBy="search-form-description"
+                    shouldShowFilters={query !== ''}
+                    worksRouteProps={null}
+                    workTypeAggregations={[]}
+                    aggregations={undefined}
+                  />
+                )}
+              </div>
+            </div>
+          </div>
+        </Space>
+
+        {results?.type === 'ResultList' && results.results.length > 0 && (
+          <>
+            <Space v={{ size: 'l', properties: ['padding-top'] }}>
+              <div className="container">
+                <div className="grid">
+                  <div
+                    className={classNames({
+                      [grid({ s: 12, m: 12, l: 12, xl: 12 })]: true,
+                    })}
+                  >
+                    <div className="flex flex--h-space-between flex--v-center">
+                      <ImagesPagination
```

Comment:
Not really sure what this abstraction is getting us?

Reply:
Felt a little bit tidier since the component's on the page twice.